client/dead.lua

```diff
@@ -1,68 +1,44 @@
-local hold = 5
 deathTime = 0
 emsNotified = false
 
--- Functions
-function OnDeath()
-    if not isDead then
-        isDead = true
-        TriggerServerEvent("hospital:server:SetDeathStatus", true)
-        TriggerServerEvent("InteractSound_SV:PlayOnSource", "demo", 0.1)
-        local player = PlayerPedId()
+local function playDeadAnimation(player)
+    if cache.vehicle then
```

Comment:
Reverse this if statement to reduce nesting

Reply:
Bit confused by what you mean. We could do a return in the first if and get rid of the else, but I don't that would improve the readability here.

Reply:
I didn't see the else, it's fine then